smwe_rom/src/compression/lc_lz2.rs

```diff
@@ -43,24 +35,21 @@ pub fn decompress(input: &[u8]) -> Result<Vec<u8>, DecompressionError> {
             break;
         }
         in_it = &in_it[1..];
-        let command = chunk_header >> 5;
-        let length = chunk_header & 0b11111;
 
-        let mut command = Command::try_from(command).map_err(|_| LcLz2Error::Command(command))?;
-        let mut length = length as usize + 1;
-
-        if let Command::LongLength = command {
-            let real_command_bits = (chunk_header >> 2) & 0b111;
-            command =
-                Command::try_from(real_command_bits).map_err(|_| LcLz2Error::LongLengthCommand(real_command_bits))?;
-            let length_part_1 = chunk_header & 0b11;
-            let length_part_2 = *in_it.first().ok_or(LcLz2Error::LongLength)?;
-            length = (((length_part_1 as usize) << 8) | (length_part_2 as usize)) + 1;
+        let (command, length) = if chunk_header & 0xe0 == 0xe0 {
```

Comment:
I'd change the `0xe0` to `0xE0` since we would like to keep our hex number literals consistent across the codebase.